joshprzybyszewski/cribbage
# Purge tailwind

## What broke / What you're adding
Removed tailwind css from our application. We made the choice to go with Material some time ago because it provides significantly more out-of-the-box functionality and doesn't require extra build steps and junk.

I also made the TODO --> TODONE to add a card back image. I grabbed an SVG from wikimedia commons.

## How you did it
I replaced all the old tailwind classes with Material's implementation of css-in-js (which is [JSS](https://cssinjs.org/?v=v10.6.0)). There are some things that will look slightly different (alerts are pretty a bit different in terms of size and font; colors are slightly different on disabled cards), but everything is generally the same. I think we may want a follow-up to make alerts look less bad, but I'm not too worried about it right now.

## How to test it and how to try to break it
Boot up the server and client. Play through part of a game. Trigger some alerts (easy ones are trying to sign in a user that doesn't exist or enter the wrong number of points for a hand). See how things look!
